apollo-http-cache/build.gradle.kts

```diff
@@ -4,6 +4,7 @@ plugins {
 
 dependencies {
   api(groovy.util.Eval.x(project, "x.dep.okHttp.okHttp"))
+  api(groovy.util.Eval.x(project, "x.dep.okio"))
```

Comment:
Jus curious: doesn't okhttp already has okio as an `api` dependency? Doesn't harm to add it again but that feels superfluous

Reply:
I think IJ couldn't resolve the imports - but I will check again.

Reply:
No you're right it works, I guess it was a fluke :)